var/spack/repos/builtin/packages/fms/package.py

```diff
@@ -104,6 +104,7 @@ class Fms(CMakePackage):
     depends_on("netcdf-fortran")
     depends_on("mpi")
     depends_on("libyaml", when="+yaml")
+    depends_on("llvm-openmp", when="+openmp %apple-clang@15:", type=("build", "run"))
```

Comment:
Shouldn't this be for all versions of apple-clang? See https://github.com/JCSDA/spack/blob/5254babe535e59397361d5f0e11f73d1e5613bc6/var/spack/repos/builtin/packages/fms/package.py#L122

Reply:
Done.